### PR TITLE
Improvement/bootstrap4

### PR DESCRIPTION
**Pull request by nandomoreirame**
This pull request is for issue #32 - **Website structure to bootstrap 4**

### What was done?

- Installed Bootstrap 4 beta and removed primer-css
- Create _ve-bootstrap.scss and _ve-bootstrap-variables.sass bootstrap files using only functions, variable, mixins and grid
- Creating a partial for the head to be shared between the layouts default.html and home.html

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     },
     "author": "vue-es.org team",
     "dependencies": {
+        "bootstrap": "4.0.0-beta",
```
Comment (coderdiaz) [inline package.json]: This dependency is not necessary check the Gemfile

Comment (nandomoreirame) [reply package.json]: The way I worked did not find Bootstrap files installed by gemfiles

Comment (nandomoreirame) [reply package.json]: unless you want to load the entire bootstrap

Comment (coderdiaz) [reply package.json]: At the moment is not priority reduce size of css file

Comment (coderdiaz) [reply package.json]: Github Pages cannot compile `scss` with external sources from npm

Comment (nandomoreirame) [reply package.json]: humm, okay, I fix it

Comment (coderdiaz) [reply package.json]: Check it, https://github.com/twbs/bootstrap-rubygem

**Comment by nandomoreirame** [comment]
@coderdiaz I think I fixed this, and added some HTML class :)

**Comment by coderdiaz** [comment]
@nandomoreirame Thanks 👍

**Comment by coderdiaz** [comment]
:shipit:

**Comment by nandomoreirame** [comment]
Thanks @coderdiaz ... Next step, improve layout

**Pull request merged**